app/controllers/greetings_controller.rb

```diff
@@ -0,0 +1,8 @@
+class GreetingsController < ApplicationController
+  def index; end
+
+  def show
+    @greeting = Greeting.all.sample
+    render json: @greeting
```

Comment:
*  I get this error in the console while I run the frontend of your app, Kindly fix this as no random greeting is being rendered. The backend API works fine as it renders random greetings 
![image](https://user-images.githubusercontent.com/52497246/211714734-9b264f94-8a31-4df1-9f5c-e1e14d54f403.png)
![image](https://user-images.githubusercontent.com/52497246/211714989-a3c0bb58-895c-4176-864b-02dd362897f8.png)

Reply:
the error was corrected and update in the react frontend side I add the screen shoot of my browser

![image](https://user-images.githubusercontent.com/92118813/211874684-fae0fc93-2beb-4c74-8e37-9f53a280261c.png)